db_stress_tool/db_stress_common.cc

```diff
@@ -233,6 +233,15 @@ size_t GenerateValue(uint32_t rand, char* v, size_t max_sz) {
   return value_sz;  // the size of the value set.
 }
 
+std::string NowNanosStr() {
+  uint64_t t = db_stress_env->NowNanos();
+  std::string ret;
+  PutFixed64(&ret, t);
+  return ret;
+}
+
+std::string GenerateTimestampForRead() { return NowNanosStr(); }
```

Comment:
Should db_stress just use `NowNanoStr()` instead?

Reply:
I imagine we will in the future add an option `-read_with_latest_ts`. Currently, we always read from latest ts. In the future, this can be set to false and we will get a random ts for read. Therefore, I think it is better to put the entire logic in a function `GenerateTimestampForRead()`.